github/feeds.go

```diff
@@ -0,0 +1,69 @@
+// Copyright 2016 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+// FeedsService provides access to the feeds related functions
+// in the GitHub API.
+//
+// GitHub API docs: https://developer.github.com/v3/activity/feeds/
+type FeedsService struct {
+	client *Client
+}
+
+// AtomRef represents an Atom reference.
+type AtomRef struct {
```

Comment:
AtomRef isn't really the best name for this.  While yes, they are being used to a reference an atom feed in this context, what GitHub is *really* doing here is using [HAL](http://stateless.co/hal_specification.html) style links.  So really just calling this `Link` would be appropriate.  However, in the interest of not using such a generic term for such a specific use case, maybe `FeedLink` is better.